boards/arm/rddrone_fmuk66/rddrone_fmuk66.dts

```diff
@@ -363,6 +363,16 @@ zephyr_udc0: &usbotg {
 		status = "okay";
 	};
 
+	fxas21002@1 {
+		compatible = "nxp,fxas21002";
```

Comment:
this breaks bisectability: you first need to make driver/bindings changes, then update boards.

Reply:
@gmarull board and binding changes are part of the same commit and driver changes are in the next commit
So, shall we make driver changes as first commit and board and bindings as next?

Reply:
Valid sequences could be: dts -> driver -> board, or dts+driver -> board

Reply:
Acknowledged !

Reply:
@gmarull .. We have made the change as per your suggestion.. Kindly re-review